src/components/Person/Person.jsx

```diff
@@ -1 +1,12 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => (
+  <>
+    <h2 className="Person__name">My name is {person.name}</h2>
+    {person.age && <p className="Person__age">I am {person.age}</p>}
+    <p className="Person__partner">
+      {person.isMarried === true ? person.partnerName : 'I am not married'} is
+      my {person.sex === 'f' ? 'husband' : 'wife'}
```

Comment:
```suggestion
      my {person.sex === FEMALE ? 'husband' : 'wife'}
```
It's better to use constants